lib/sidekiq/xaclient.rb

```diff
@@ -0,0 +1,52 @@
+require "sidekiq/client"
+
+begin
+  require "after_commit_everywhere"
+rescue LoadError
+  Sidekiq.logger.error("You need to add after_commit_everywhere to your Gemfile for this to work")
+  exit(-127)
+end
+
+module Sidekiq
+  module XAClient
```

Comment:
What's the advantage of introducing "industry jargon" to the codebase if it needs to be explained in the PR description?